### DISCUSSION TITLE
Error Message after updating next.config.js

### DISCUSSION THREAD

**Discussion by Mona904** [neuefische/questions#596] [Web] [answered]
Hello all, 
I have updated the code as requested but now I am getting an server error message that says "Error: Image with src https://www.picturethisai.com/wiki-image/1080/1 is missing required "width" property". 

 🧮 Pull Request or link to code:
https://github.com/Mona904/capstone-project/pull/4

Please advise on how to proceed.
Thanks

**Comment by Roland-Hufnagel** [comment, marked as answer]
Hi Mona, as you can read in the documentation from 'next-image' or in the handout you have to give the `Image`-component a `width`and a `height`-property. Have a look here: 
https://nextjs.org/docs/pages/api-reference/components/image
